### PR TITLE
Adds Gun Cabinets

### PR DESCRIPTION
**Pull request by Xantholne**
Ports Guncabinet design from Paradise (prob ported from somewhere else?). Features several preset cabinets and closets with various rigs for any admin gimmick sort of stuff.

:cl: Xantholne
add: Admins can now spawn new empty and preset gun cabinets for any sort of gimmick or event.
/:cl:

### CODE REVIEW

**Comment by SmokingKilz** [comment]
While it's nice to have this as an admin spawn, I find this has most value for mapping.

**Comment by Xantholne** [comment]
I agree. I believe we can use these for mapping later on after the combat overhaul. 
Or even on the new station as a new armory.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -341,3 +344,118 @@
...
+	new /obj/item/clothing/mask/gas/sechailer/swat(src)
+	new /obj/item/storage/box/flashbangs(src)
+	new /obj/item/shield/riot/tele(src)
+	new /obj/item/storage/belt/security/full(src)
+	new /obj/item/gun/energy/e_gun/hos(src)
+	new /obj/item/device/flashlight/seclite(src)
+	new /obj/item/pinpointer/nuke(src)
+
+/obj/structure/closet/secure_closet/warden/enforcer
+	name = "\proper warden's locker"
```
Comment (FlattestGuitar) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: remove the `\proper`

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 2]
```diff
@@ -341,3 +344,118 @@
...
+
+/obj/structure/closet/secure_closet/guncabinet/ar/PopulateContents()
+	..()
+	for(var/i in 1 to 6)
+		new /obj/item/ammo_box/magazine/m556(src)
+	for(var/i in 1 to 3)
+		new /obj/item/gun/ballistic/automatic/ar(src)
+
+/obj/structure/closet/secure_closet/hos/enforcer
+	name = "\proper head of security's locker"
```
Comment (FlattestGuitar) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: same

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 3]
```diff
@@ -172,6 +172,9 @@
 
 /obj/structure/closet/secure_closet/security/sec/PopulateContents()
 	..()
+	new /obj/item/gun/ballistic/automatic/pistol/enforcer
```
Comment (FlattestGuitar) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: You just casually added enforcers for all security officers.

Pls no.

Comment (Xantholne) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: These aren't the same lockers as the regular ones that currently spawn

**Comment by SamHPurp** [comment]
I need to look into it properly, but I'm not immediately a fan of the concept.

**Comment by FlattestGuitar** [comment]
Again, the sprites need to be changed and put in a new folder in the code. Please do that.

Also, @SamHPurp, this is admin-only. Might be useful for events or something. I don't see why not, since you can't add locker sprites at runtime.

**Comment by SamHPurp** [comment]
@FlattestGuitar read @SmokingKilz 's comment.

**Comment by SmokingKilz** [comment]
I mean, @FlattestGuitar made some valid requests for changes. I don't see what would be the issue with new lockers though, especially if players cannot make them and it's just bound for admins right now. The sprites in question look rather nice if you ask me.

**Pull request closed without merging**